[quant] Quantizable documentation

HDCharles opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #71192
* __->__ #71191
* #71190
* #71189



Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

HDCharles committed:
[quant] Quantizable documentation

Minor documentation entry for the quantizable LSTM and MHA classes.
ghstack-source-id: 146629295
ghstack-source-id: 146875020

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant] Quantizable documentation"


Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant] Quantizable documentation"


Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant] Quantizable documentation"


Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant] Quantizable documentation"


Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

vkuzo commented on docs/source/quantization.rst:
```diff
@@ -39,7 +39,106 @@ APIs are provided that incorporate typical workflows of converting FP32 model
 
 Quantization API Summary
------------------------------
+------------------------
```
nit: is this new code intended to be in the PR?

HDCharles replied:
no

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

z-a-f reviewed (approved):
lg -- one nit: Should we explain what the "quantizable" means, and why it's needed as part of the documentation?

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284/)

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33452284).

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.


Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant] Quantizable documentation"




Minor documentation entry for the quantizable LSTM and MHA classes.

Differential Revision: [D33452284](https://our.internmc.facebook.com/intern/diff/D33452284)

[ghstack-poisoned]
